onnxruntime/test/optimizer/initializer_test.cc

```diff
@@ -80,7 +80,8 @@ TEST(OptimizerInitializerTest, LoadExternalData) {
 
         if (offset + length <= tensor_data_span.size()) {
           Initializer i(tensor_proto, tensor_data_dir_path);
-          EXPECT_EQ(gsl::make_span(i.data<int32_t>(), i.size()), tensor_data_span.subspan(offset, length));
+          const auto* i_data = i.data<int32_t>();
```

Comment:
could be clearer:
`gsl::span<const int> i_data_span{i.data<int32_t>(), i.size()};`